### PR TITLE
Add backticks in comments where required

### PR DESCRIPTION
**Pull request by thisisnotashwin**
Changes proposed in this PR:
- Add backticks to helm comments.

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -1040,7 +1040,7 @@ ui:
     provider: "prometheus"
 
     # baseURL is the URL of the prometheus server, usually the service URL.
```
Comment (ndhanushkodi) [inline values.yaml:1042]: L1038 also needs ui.enabled in backticks

File: values.yaml [hunk 2]
```diff
@@ -1040,7 +1040,7 @@ ui:
...
-    # This value is only used if ui.enabled is set to true.
+    # This value is only used if `ui.enabled` is set to true.
     # @type: string
     baseURL: http://prometheus-server
```
Comment (ndhanushkodi) [inline values.yaml:1045]: Also L1304 needs prometheus.io/path in backticks

**Review by ndhanushkodi — APPROVED** [review]
🎉

File: values.yaml (hunk 1 above)
Comment (thisisnotashwin) [reply values.yaml:1042]: good catch!!

File: values.yaml (hunk 2 above)
Comment (thisisnotashwin) [reply values.yaml:1045]: good catch!!

**Pull request merged**